examples/titanic/README.md

```diff
@@ -70,13 +70,25 @@ python assets/algo_random_forest/algo.py train \
   --debug \
   --opener-path assets/dataset/opener.py \
   --data-samples-path assets/train_data_samples \
-  --output-predictions-path assets/pred-train.csv
-  
+  --output-model-path assets/model/model \
+  --log-path assets/logs/train.log
+
+python assets/algo_random_forest/algo.py predict \
+  --debug \
+  --opener-path assets/dataset/opener.py \
+  --data-samples-path assets/train_data_samples \
+  --output-predictions-path assets/pred-train.csv \
+  --models-path assets/model/ \
+  --log-path assets/logs/train_predict.log \
+  model
+
 python assets/objective/metrics.py \
   --debug \
   --opener-path assets/dataset/opener.py \
   --data-samples-path assets/train_data_samples \
-  --input-predictions-path assets/pred-train.csv
+  --input-predictions-path assets/pred-train.csv \
+  --output-perf-path assets/perf-train.json \
+  --log-path assets/logs/train_metrics.log
```

Comment:
As far as I understand the prediction and the score should not be part of the training task?

Reply:
The training score can be interesting, but it's not automatically computed anymore.